packages/components/src/Workflow/_docs/Workflow.mdx

```diff
@@ -22,19 +22,19 @@ import * as WorkflowHeader from "./WorkflowHeader.stories"
 
 ## Overview
 
-This is a page template component containing the header, footer and main landmarks that compose a Workflow page. Its purpose is to guide an customer through a multi-step form to create a Workflow.
+This is a page template component containing the header, footer, and main landmarks that compose a Workflow page. Its purpose is to guide a customer through a multi-step form to create a Workflow.
 
-The Worflow is intended as a page wrapper and sets minimum height of `100vh` with a sticky `Header` component. The `Children` will be wrapped in a unstyled main landmark to provide flexibility for inner content layouts.
+The Worflow is intended as a page wrapper and sets a minimum height of `100vh` with a sticky `Header` component. The `Children` will be wrapped in an unstyled main landmark to provide flexibility for inner content layouts.
 
-This component has responsiveness baked in to support 400% zoom and smaller screen sizes. This will also un-stick the `Header` to provide more realestate for the user.
+To ensure at readability at 400% zoom (See WCAG's [Reflow criteria](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html)) this component will collapse to a vertical layout on smaller screen sizes and remove `sticky` positioning to maximise screen real estate. You can see an example in [the responsive workflow story](/story/pages-workflow-components-workflow--responsive-workflow).
```

Comment:
I've linked out to a story instead of creating a Canvas in the MDX since the `defaultViewport` size that should be inherited in the `of={Workflow.ResponsiveWorkflow}` story did not carry across.